### PR TITLE
tests for AMKO boot up checks and leader field change in AMKOCluster

### PR DESCRIPTION
**Pull request by monotosh-avi**
Adding tests to check
- updating isLeader field in AMKOCluster object leads to shut down of API server
- AMKO should not bbot up if Both Member clusters are leaders
- AMKO should boou up successfully, if one member Cluster is leader and another is follower
- AMKO should boot up even if one member cluster is braught down by shutting down cluster with envtest

### CODE REVIEW

File: gslb/test/bootuptest/bootup_test.go [hunk 1]
```diff
@@ -0,0 +1,438 @@
+package bootuptest_test
```
Comment (sudswasavi) [inline gslb/test/bootuptest/bootup_test.go:1]: missing copyright?

**Review by sudswasavi — COMMENTED** [review]
looks great.

File: gslb/test/bootuptest/bootup_test.go (hunk 1 above)
Comment (monotosh-avi) [reply gslb/test/bootuptest/bootup_test.go:15]: Missed it, added now. thanks.

**Review by sudswasavi — DISMISSED** [review]

File: helm/amko/crds/amko_cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+apiVersion: apiextensions.k8s.io/v1
```
Comment (hemantavi) [inline helm/amko/crds/amko_cluster.yaml:1]: this file is already added with a different name. Can you please remove it?

**Review by hemantavi — APPROVED** [review]

**Pull request merged**